cli/assets/templates/rust/.cargo/config.toml

```diff
@@ -10,8 +10,8 @@ rustflags = [
     "-C", "link-arg=--initial-memory=65536",
     "-C", "link-arg=--max-memory=65536",
 
-    # Reserve 1024 bytes of stack space, offset from 6580
-    "-C", "link-arg=-zstack-size=7584",
+    # Reserve 1228 bytes of stack space, offset from 6580
+    "-C", "link-arg=-zstack-size=7808",
```

Comment:
Should we bump this to 2048+ bytes just to be safe?

Reply:
I do not think that there's a perfect number or at least I do not know a 
I think that we should just add a comment that suggests to bump this value in case the framebuffer gets corrupted.